packages/federation/test/federation-compatibility.test.ts

```diff
@@ -71,6 +75,9 @@ describe('Federation Compatibility', () => {
         );
         const sortedInputSchema = lexicographicSortSchema(filteredInputSchema);
         const sortedStitchedSchema = lexicographicSortSchema(stitchedSchema);
+        if (supergraphName === 'non-resolvable-interface-object') {
+          return;
```

Comment:
Leave a comment explaining why we're skipping this supergraph.

Reply:
Done!